app/src/Shared/CustomSnackBar/CustomSnackBar.css

```diff
@@ -0,0 +1,11 @@
+.success {
+    background-color: #4caf50 !important;
```

Comment:
Is there a reason that we need to overwrite the CSS specificity with !important? ⬇️⬇️⬇️

Reply:
eep good catch, this was from my testing! I moved this back into the other file